langchain/chains/base.py

```diff
@@ -33,6 +33,18 @@ def save_context(self, inputs: Dict[str, Any], outputs: Dict[str, str]) -> None:
     def clear(self) -> None:
         """Clear memory contents."""
 
+    def hydrate(
```

Comment:
Just my two cents: "hydrate" is not an intuitive word for this method. I'd prefer something like `initialize_memory` or `jog_memory`.